tools/build_pytorch_libs.py

```diff
@@ -73,7 +73,14 @@ def cmake_defines(lst, **kwargs):
     import ninja
     USE_NINJA = True
 except ImportError:
-    USE_NINJA = False
+    # The ninja package is different in Anaconda Cloud and PYPI. The one in Anaconda Cloud
```

Comment:
If we use `which` to find ninja, then we can delete the `import ninja` check. We do not use the ninja python packages anymore.

Reply:
Sure!